bin/webpack.js

```diff
@@ -86,6 +86,16 @@ if (!cli.installed) {
 		input: process.stdin,
 		output: process.stderr
 	});
+
+	// If we're not in terminal mode, readline won't execute the callback function below.  Return here
+	// so that we can set the exit code properly.
+	if (!questionInterface.terminal) {
+		console.error(
+			"You need to install 'webpack-cli' to use webpack via CLI.\n" +
+				"You can also install the CLI manually."
+		);
+		process.exit(1);
```

Comment:
```suggestion
		process.exitCode = 1;
		return;
```

`process.exit` may not flush console message so they could get lost.

Reply:
I had that initially, because I just copy/pasted from the question callback, but the linter complained that you can't `return` when you're not in a function.